tests/test_redshiftdata/test_redshiftdata.py

```diff
@@ -0,0 +1,82 @@
+import boto3
```

Comment:
Woul dyou midn extending the tests here to also include positive tests, i.e. using the boto3-client to cancel/describe/execute a statement?
I know the functionality is covered in `test_server.py`, but the majority of the users will use the boto3-client to call these methods - so we should have test cases to verify it works with boto3.

Reply:
Sure, I added some positive tests as well.